adapta/storage/models/enum.py

```diff
@@ -0,0 +1,13 @@
+"""
+ Storage enums.
+"""
+
+from enum import Enum
+
+
+class QueryEnabledStoreOptions(Enum):
+    """
+    QES options aliases in Adapta.
```

Comment:
Maybe describe the related options object(s) here.
E.g. if `CONCAT_OPTIONS` is used, `MetaFrameOptions` is expected